src/Contao/Widgets/MultiColumnWizard.php

```diff
@@ -204,9 +204,7 @@ class_alias(self::class, 'MultiColumnWizard');
 
         // Add symfony translator.
         if ((bool) ($arrAttributes['useTranslator'] ?? false)) {
-            $translator = System::getContainer()->get('translator');
-            assert($translator instanceof TranslatorInterface);
-            $this->translator = $translator;
+            $this->translator = System::getContainer()->get('translator');
```

Comment:
Can't do this here, the return value of `ContainerInterface::get` is type object.
Therefore the assertion is correct and needed.

Assertions are good and have no downside in production (as they are set to `-1` on correctly configured (See: [php.net](https://www.php.net/manual/en/ini.core.php#ini.zend.assertions)).